__tests__/integration/txLogger.js

```diff
@@ -0,0 +1,137 @@
+import * as Path from "path";
+
+const fsold = require('fs');
+const fs = fsold.promises;
+
+export const loggers = {
+  /**
+   * @type: TxLogger
+   */
+  test: null
+}
+
+/**
+ * A logger for every transaction on the integration tests for debugging.
+ */
+export class TxLogger {
+  #instanceFilename;
+  #fileFullPath;
+
+  get filename() {
+    return this.#instanceFilename;
+  }
+
+  /**
+   * Builds the log filename based on current time and an optional title
+   * @param {string} [title] Optional title. Keep it short and simple for readability
+   */
+  constructor(title) {
+    const date = new Date();
+    const timestamp = date.toISOString()
```

Comment:
You can get the timestamp with `Math.floor(Date.now() / 1000);`

Reply:
Renamed this variable `humanReadableTimestamp`. One example of a filename created with this constructor is
```20220224T084737-title-integrationTest.log```

Usage of the native timestamp would not be interesting here, as an _integer_ is not intuitive for a reader to understand when the file was created. E.g.: `1645738340-title-integrationTest.log`

The docstring was also updated to clarify this process.

Reply:
I agree about the readability. I misunderstood because of the var name